chart/kubeapps/templates/kubeops-config.yaml

```diff
@@ -10,5 +10,5 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   additional-clusters.conf: |-
```

Comment:
should this be `clusters.conf`?

Reply:
It will be updated (and the related deployment template, kubeops args, etc.) , but I've left that for when I get to #1942 . It was important in this PR to do the chart change mainly - I included the frontend changes only because the multicluster config was initially passed through directly as part of the feature flag, otherwise it wasn't really special.